# For internal communication of exception types.

uint8 NONE = 0
uint8 TIMEOUT_EXCEPTION = 1
uint8 TRANSFORM_EXCEPTION = 2
uint8 INTERNAL = 3
uint8 OTHER = 4
